scripts/start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+function launch {
+	cd `dirname $1`
+	./`basename $1` &
+	cd -
+	echo "$1 has been launched"
+	sleep 5
+}
+
+set +e
+
+
+launch cmd/core-metadata/core-metadata
```

Comment:
maybe I am missing something, but I thought that built artefacts are kept in separate dir (`build` dir for example)

Reply:
It worked fine with export microservices, but core services must have a config file to be able to run. So the simplest solution is to generate the executable in cmd/microservice, so it can be run from that directory where the config file is stored. 
I think we can discuss this in a https://github.com/edgexfoundry/edgex-go issue better.